src/02-data-types.js

```diff
@@ -36,7 +40,9 @@ function convertToNumber(val) {}
  *  convertToString(99);
  *  //> "99"
  */
-function convertToString(val) {}
+function convertToString(val) {
+  return val.toString()
```

Comment:
Much like our `charAt()` call we can shorten this to ` val + "" ` and JS will implicitly convert this to a string.